src/components/Bookmark.js

```diff
@@ -1,23 +1,27 @@
 import styled from 'styled-components/macro'
 
+export default function Bookmark({ onClick, isActive }) {
+  return (
+    <Wrapper isActive={isActive} onClick={onClick}>
```

Comment:
Maybe add an `aria-Label` so that the button is even more accessible. 
```suggestion
    <Wrapper isActive={isActive} onClick={onClick} aria-label="Bookmark">
```